app/src/main/java/cm/aptoide/pt/search/view/SearchResultFragment.java

```diff
@@ -88,6 +91,8 @@
       "followed_stores_search_list_state";
   @Inject AnalyticsManager analyticsManager;
   @Inject NavigationTracker navigationTracker;
+  @Inject FragmentNavigator fragmentNavigator;
+  @Inject SearchResultPresenter searchResultPresenter;
```

Comment:
Analytics, NavigationTracker and FragmentNavigator are not used anymore.

Reply:
Fixed